susemanager-utils/susemanager-sls/salt/packages/pkginstall.sls

```diff
@@ -7,7 +7,9 @@ pkg_installed:
 {%- endif %}
     -   pkgs:
 {%- for pkg, arch, version in pillar.get('param_pkgs', []) %}
-    {%- if grains.get('__suse_reserved_pkg_all_versions_support', False) %}
+    {%- if grains['os_family'] == 'Debian' %}
+        - {{ pkg }}:{{ arch }}: {{ version }}
```

Comment:
Is it intentional that in Debian `pkg` and `arch` are separated by colon while in RPM-based distros they are separated by dot?

Reply:
Yes, it's intentional. It was a constraint from Salt side. I believe it was because debian package names are allowed to have dots in them.

See: https://github.com/openSUSE/salt/commit/2e9029db72879b30979a59e057d8712399ef567f#diff-f28fd94bb04c597a37eaea3b885be3bcR222